thorlcr/graph/thgraphslave.cpp

```diff
@@ -1775,14 +1775,34 @@ IReplicatedFile *createEnsurePrimaryPartFile(CActivityBase &activity, const char
 class CFileCache;
 class CLazyFileIO : public CInterface, implements IFileIO, implements IDelayedFile
 {
+    class CFileIOWrapper : public CSimpleInterfaceOf<IFileIO>
```

Comment:
Is this change required for this issue or is it something else you spotted needing fixing?